README.rst

```diff
@@ -131,142 +111,24 @@ Documentation
 | Stable  | |stable|    | `Documentation of the stable (i.e. most recent release) branch. <https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/>`_ |
 +---------+-------------+------------------------------------------------------------------------------------------------------------------------------------------+
 
-Tutorials
----------
-A great way to start with NeMo is by checking `one of our tutorials <https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/starthere/tutorials.html>`_.
-
-Getting help with NeMo
-----------------------
-FAQ can be found on NeMo's `Discussions board <https://github.com/NVIDIA/NeMo/discussions>`_. You are welcome to ask questions or start discussions there.
-
 
 Installation
 ------------
 
-Conda
-~~~~~
-
-We recommend installing NeMo in a fresh Conda environment.
-
-.. code-block:: bash
-
-    conda create --name nemo python==3.8
-    conda activate nemo
-
-Install PyTorch using their `configurator <https://pytorch.org/get-started/locally/>`_. 
-
-.. code-block:: bash
+https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/stable/starthere/intro.html#installation
 
-    conda install pytorch torchvision torchaudio cudatoolkit=11.3 -c pytorch
-
-.. note::
-
-  The command used to install PyTorch may depend on your system.
```

Comment:
Instead of getting rid of the installation from the README, how about we make a dropdown?


like this:


<details>
<summary>Pip</summary>

```bash
apt-get update && apt-get install -y libsndfile1 ffmpeg
pip install Cython
pip install nemo_toolkit['all']
```

</details>

Reply:
Main thing - I wanted to have the instructions in one place rather than two. I like idea of dropdown, not sure if it is possible for rst, but I'll try

Reply:
not sure how to add dropdown in rst. Any pointers?

Reply:
Ah, did not realise it was an RST file, nevermind!